structure-proposal.md

```diff
@@ -0,0 +1,104 @@
+    
+- This structure can be adapted and augmented by explicitizing additional, critical steps that are particularly pertinent to and well-suited for a SEM context (Paxton et al., 2001):
+    1. developing a theoretically derived research question of interest (= verbal description)
+        
+        - e.g. examination of goodness-of-fit statistics under varying degrees of misspecification
+        - comparison of the maximum likelihood (ML) to (2SLS)
+        
+    2. creating a valid model
+        
+        Modularities:
+        - Structure: (e.g. CFA or SEM)
+        - Size: number of latents and indicators
+        - Complexity: (cross-loaded indicators, Reciprocal paths, Exogenous predictors)
+        → select target model for (assumed) *general* model types
+        
+    3. designing specific experimental conditions (partly derived from 1)
+    Common (general) modularities:
+    - almost universally: sample size
+    - distribution of the observed variables
+    - estimation method
+    - extent of misspecification (and to omit or include misspecification paths)
+    4. choosing values of population
+        
+        Modularities:
+        - should reflect values commonly encounterd in applied research.
+        
+    5. choosing an appropriate software package (optional, could be standardized for enhanced evaluation)
+    6. executing the simulations
+        
+        Modularities:
+        - inclusion / exclusion of “imperfect” samples (non converged / improper solutions)
+        
+    7. file storage (to be standardized, just kept here for completionism)
+    8. troubleshooting and verification (to be standardized, just kept here for completionism)
+    9. summarizing results
+        
+        Modularities:
+        - descriptive
+        - graphical
+        - inferential
+        
+
+Here steps 1, 3, 6 and 9 correspond to Harwell et al. (1996). 2 and 4 are especially relevant for an SEM context and should thus be included explicitly. Note the step “file storage” was excluded here and should be (pre-)standardized across collaborators to enhance post-hoc evaluation. Note critically step 6 involves the decision of inclusion or exclusion of “imperfect” samples (non converged / improper solutions).
+
+Other structures identified match these steps on a rather abstract level and do not add any modularity nor do they promise increased structural standardization in simulations in the context of SEM (Law & Comas, 1991; Shannon, 1992; Smith & Marshall, 2010). Any further abstraction of the structure to fit a general concept of simulations will not benefit but rather introduce potential blur to the current stage of development of the AC approach in a SEM simulation setting. 
+
+Note. Burton et al. (2006) proposes a separate step of Criteria to evaluate the performance of statistical methods for different scenarios which is subsumed here in step 4 designing specific experimental conditions.
```

Comment:
Why do you think this is the right thing to do? I would disagree. It might be, that there is a very close connection between method choice and evaluation criteria, but it does not have to be! If the latter is true, selecting method & evaluation criteria could be a source of bias that could be seperated.

Reply:
I agree with this.